python/cac_tripplanner/cms/views.py

```diff
@@ -56,3 +60,26 @@ def tips_and_tricks_detail(request, slug):
     context = RequestContext(request, {'article': tips_and_tricks})
     return render_to_response('tips-and-tricks-detail.html',
                               context_instance=context)
+
+class AllArticles(View):
+    """ API endpoint for the Articles model """
+
+    def get(self, request, *args, **kwargs):
+        """ GET title, URL, and images for the 20 most recent articles that are published"""
+        results = Article.objects.published().order_by('-publish_date')[:20]
```

Comment:
Hm, yeah.  The featured destinations also have a hard-coded 20 limit to them; not sure if we might want to paginate, or what.